plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -91,6 +91,11 @@ public static DistributedQueryRunner createIcebergQueryRunner(Map<String, String
         return queryRunner;
     }
 
+    public static Path getFileHiveMetastoreDirectory(DistributedQueryRunner queryRunner)
+    {
+        return queryRunner.getCoordinator().getBaseDataDir().resolve("iceberg_data");
+    }
```

Comment:
This change looks unrelated. I would normally not change unrelated files to avoid commit conflicts, but I'm not sure how that works in the Trino community.

Reply:
I only extracted it out because the tests needed access to this Path, otherwise yes I'd agree.